### ISSUE TITLE
Thalorien Dawnseeker

### ISSUE THREAD

**Issue by giovannilol88** [dalaranwow/dalaran-wow#5905]
[//]: # (INFORMATION ABOUT TAGS)Thalorien Dawnseeker
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. i cant complete Quest Thalorien Dawnseeker
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

![image](https://user-images.githubusercontent.com/67504419/85922554-9a2c2680-b884-11ea-9a4b-8527394a3a9f.png)

**Comment by Golganeth** [comment]
Abandon and retake the quest, and/or reset your objectives. As posted in #5903, this quest works just fine, several players have completed it during the last days.
Closing up

**Comment by giovannilol88** [comment]
Its okay Azeroth GM help . I honor he

Il Sab 27 Giu 2020, 16:18 Golganeth <notifications@github.com> ha scritto:
